test/korma/test/core.clj

```diff
@@ -738,6 +766,14 @@
               "dry run :: SELECT \"address3\".* FROM \"address3\" WHERE (\"address3\".\"user3_id\" = ?) :: [1]\n")
          (with-out-str (dry-run (select user3 (with address3)))))))
 
+(defentity address3-ck (transform identity))
+(defentity user3-ck (pk :id :id2) (has-one address3-ck (fk :uid :uid2)))
+
+(deftest test-has-one-with-transform-fn-for-subentity-composite-key
```

Comment:
test composite-key handling with has-one relationship and transform fn for subentity